src/pages/CartPage/types.ts

```diff
@@ -0,0 +1,48 @@
+export type CartProduct = {
+  id: number;
+  title: string;
+  price: number;
+  image: string;
+  amount: number;
+};
+
+export enum ActionType {
+  add = "add",
+  remove = "remove",
+  increaseAmount = "increase-amount",
+  decreaseAmount = "decrease-amount",
```

Comment:
Basically, you've created 4 constants here. And for constant names usually upper case is used

 ```suggestion
  ADD = "add",
  REMOVE = "remove",
  INCREASE_AMOUNT = "increaseAmount",
  DECREASE_AMOUNT = "decreaseAmount",
```